pages/cockpit/user-guide/installation-health-check.adoc

```diff
@@ -0,0 +1,28 @@
+= Installation Health Check
+:page-sidebar: cockpit_sidebar
+:page-permalink: cockpit/1.x/cockpit_userguide_installation_health_check.html
+:page-folder: cockpit/userguide
+:page-description: Gravitee.io Cockpit - Installation health check
+:page-keywords: Gravitee.io, API Platform, API Management, Cockpit, documentation, manual, guide
+
+
+== Compatibility notice
+
+NOTE: This feature is only available with APIM & AM 3.8 or higher
+
+== Overview
+
+Installations that are registered with Cockpit will send valuable information to the dashboard, including configuration of the installation, list of nodes, plugins etc. and also some regular updates regarding their health status.
+
+On the Cockpit UI, you can view all the nodes of an installation on the Organization dashboard.
```

Comment:
Not sure I'm not an english expert but I would say it like the following.
```suggestion
Installations that are registered with Cockpit will send valuable information to the dashboard, including configuration of the installation, list of nodes, plugins, also some regular updates regarding their health status, etc.

On the Cockpit UI, you can see all the nodes of an installation on the Organization dashboard.
```
 Wdyt ?

Reply:
I would keep view, but the rest looks nicer. Thanks :-)